File: Easy/Java/1619. Mean of Array After Removing Some Elements.java
class Solution {
    public double trimMean(int[] arr) {
        int sum = 0;
        int fivePercent = arr.length / 20;
        Arrays.sort(arr);

        for(int i = fivePercent; i < arr.length - fivePercent; i++){
            sum += arr[i];
        }

        return (sum * 1.0) / (arr.length - (2 * fivePercent));
    }
}

File: Easy/JavaScript/1619. Mean of Array After Removing Some Elements.js
var trimMean = function(arr) {
    let sum = 0;
    let fivePercent = arr.length / 20;
    arr.sort((a, b) => a - b);

    for(let i = fivePercent; i < arr.length - fivePercent; i++){
        sum += arr[i];
    }

    return (sum * 1.0) / (arr.length - (2 * fivePercent));
};
